### PR TITLE
added vue-swatches package, including story in storybook

### PR DESCRIPTION
**Pull request by dominikbruechner**
# Issue: VUE-56
https://ticketsystem.schul-cloud.org/browse/VUE-56

## Description

- added external vue-swatches component
- needs to be extended for accesiblity
- aria label by is already an open issue and in work
- maybe we want to contribute https://saintplay.github.io/vue-swatches/
- component is relatively powerful, enables custom styling, different modes of display, e. g. https://saintplay.github.io/vue-swatches/#sub-inline-custom-u-i

## Screenshot

Screen Shot 2019-10-23 at 11 47 59
Screen Shot 2019-10-23 at 11 48 03

### CODE REVIEW

**Comment by dominikbruechner** [comment]
@adrianjost Should we add a unit test to an external library? - probs not...

File: stories/misc.stories.js [hunk 1]
```diff
@@ -12,11 +12,53 @@ import Searchbar from "@components/molecules/Searchbar";
...
+				"#1DE9B6",
+				"#546E7A",
+				"#FFC400",
+				"#BCAAA4",
+				"#FF4081",
+				"#FFEE58",
+			],
+		}),
+		template: `
+		<VueSwatchesColors v-model="color" :colors="colors" row-length="3" shapes="circles" show-border popover-to="right"></VueSwatchesColors>
```
Comment (adrianjost) [inline stories/misc.stories.js]: Please wrap this library in a cwrapper component.
```
<template>
  <VueSwatchesColors v-bind="$attrs" />
</template>
```

**Comment by adrianjost** [comment]
> @adrianjost Should we add a unit test to an external library? - probs not...

I think no real tests. Maybe our default tests to check if the component actually can get rendered (`it(...isValidComponent(BaseAudio));`)

**Comment by adrianjost** [comment]
@dominikbruechner By the way, when you add a wrapper component could you add the story directly next to it?
So we then have a 
```
/atoms
   colorPicker.vue
   colorPicker.stories.js
   colorPicker.unit.js
```

and in the story file use:
```
import { storiesOf } from "@storybook/vue";

import ColorPicker from "./ColorPicker";

storiesOf("Atoms|ColorPicker", module)
  .add(
	"Horizontal",
	() => ({
		components: { ColorPicker},
		template: `...`,
	}))
   .add(
	"Modal",
	() => ({
		components: { ColorPicker},
		template: `...`,
	}));
```
to create a story. This will improve our organization of stories. With everything before the `|` in the story name you can create Headings in the Sidebar, and everything after the `|` can create some subfolders. That way we can have multiple stories for one component without polutiong the sidebar. (Subfolders are by default not extended/visible)
For the BaseComponents we already started this new organization with the BaseModal
image

**Comment by dominikbruechner** [comment]
> @dominikbruechner By the way, when you add a wrapper component could you add the story directly next to it?
> So we then have a
> 
> ```
> /atoms
>    colorPicker.vue
>    colorPicker.stories.js
>    colorPicker.unit.js
> ```
> 
> and in the story file use:
> 
> ```
> import { storiesOf } from "@storybook/vue";
> 
> import ColorPicker from "./ColorPicker";
> 
> storiesOf("Atoms|ColorPicker", module)
>   .add(
> 	"Horizontal",
> 	() => ({
> 		components: { ColorPicker},
> 		template: `...`,
> 	}))
>    .add(
> 	"Modal",
> 	() => ({
> 		components: { ColorPicker},
> 		template: `...`,
> 	}));
> ```
> 
> to create a story. This will improve our organization of stories. With everything before the `|` in the story name you can create Headings in the Sidebar, and everything after the `|` can create some subfolders. That way we can have multiple stories for one component without polutiong the sidebar. (Subfolders are by default not extended/visible)
> For the BaseComponents we already started this new organization with the BaseModal
> image

yes will do
we need to adapt the hygen new component to match this pattern (todo)

**Comment by dominikbruechner** [comment]
@adrianjost done so far

**Pull request merged**